config/config.json

```diff
@@ -4,6 +4,7 @@
     "password": "123456",
     "database": "im",
     "host": "127.0.0.1",
+    "port": "35432",
```

Comment:
@WangBoxue Actually you can use the config "docker_development93", it is now nearly the same.

run the app in this way

```
NODE_ENV=docker_development93 node app.js
```